misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/core/TestAnnotationProcessor.java

```diff
@@ -137,7 +173,45 @@ public void process(FunctionNode functionNode, List<AnnotationAttachmentNode> an
                             vals[1] = value;
                         }
                     });
-                    suite.addMockFunction(vals[0] + MOCK_ANNOTATION_DELIMITER + vals[1], functionName);
+
+                    if (bLangPackage == null) {
+                        setBlangPackage(attachmentNode);
```

Comment:
Instead of trying to extract the module using the annotation, check and see if you can get it when initializing the annotation processor.

Reply:
I removed setting the bLangPackage entirely since there is already an instance of the BlangPackage that is initialized in the process() function. 

I am simply using this instance throughout the code.